### ISSUE TITLE
Support bootkube render's options for existing etcd server TLS assets

### ISSUE THREAD

**Issue by bzub** [poseidon/bootkube-terraform#13]
Similar to the current `ca_*` variables for existing k8s-tls assets. I will submit a PR if it sounds good.

Perhaps:
- `bootkube render --etcd-ca-path` -> `var.etcd_ca_path`
- `bootkube render --etcd-certificate-path` -> `var.etcd_certificate_path`
- `bootkube render --etcd-private-key-path` -> `var.etcd_private_key_path`

Without this a user has to do a lot of manual changes on apiserver secrets manifest to connect to a pre-existing etcd server specified with `var.etcd_servers`. Thanks!

**Comment by dghubble** [comment]
Thanks for the suggestion. Right now, there are a few reasons I'm not particularly in favor, but its not "no forever".

Although this project almost matches the functionality of kubernetes-incubator/bootkube, its not completely 1:1, it primarily acts as a low-level component of [Typhoon](https://github.com/poseidon/typhoon). With Typhoon, I'd like to provide the "right" amount of knobs for adjustment, but not enough to allow a user to go down a path I wouldn't recommend or support. Any of the clusters *could* use etcd in different modes, but I'd prefer folks use the etcd setup each platform comes with and consider it an implementation detail. When I worked on Tectonic, my team added support for all sorts of different ways of connecting to etcd and I've regretted that from the complexity and maintenance perspective. So I'm not saying "external etcd is crazy Bryan", I'm just saying its not a setup I wish to support, I'd prefer folks customize that on their own.

Now, there still a case that the option could be added here alone. Perhaps are you thinking of limiting this to just non-self-hosted etcd? If it were to be added, it should use `etcd_ca`, `etcd_certificate`, `etcd_key` without path - using path decides on behalf of the user that their etcd materials are files on disk, when perhaps they wish to use some other mechanism to create them.

On the other hand, I don't envision using these options (or `ca_*` for that matter) in Typhoon, so these features may not get exercised or maintained well. There is a case for dropping the lot. I've seen so many questions that boil down to folks manually creating TLS credentials (that don't work for some subtle reason), when the default auto-generates them correctly.
